packages/runtime/container-runtime/src/blobManager.ts

```diff
@@ -641,9 +617,6 @@ export class BlobManager extends TypedEventEmitter<IBlobManagerEvents> {
 			return;
 		}
 		if (!this.runtime.connected) {
```

Comment:
the pre-exisiting code here doesn't make sense to me, why do we only retry if !connected? it seems like we should always retry?

Reply:
also, i'm wondering if we are missing a check for the abort signal here, as if we aborted we don't want to retry, not sure how the code behaves

Reply:
I'm not sure that we should always retry. I believe that the intention was to only retry fail uploads when we are disconnected, but rejecting when we are connected to let user know about not network issues. Should we just retry indefinitely?

Regarding to the abortion signal, the check at the beginning of the method should be enough to intercept retries I believe.